samples/kvsWebRTCClientMasterGstSample.c

```diff
@@ -109,12 +111,12 @@ GstFlowReturn on_new_sample(GstElement* sink, gpointer data, UINT64 trackid)
 
 GstFlowReturn on_new_sample_video(GstElement* sink, gpointer data)
```

Comment:
pSampleConfiguration is already passed as data here. We can use that to pass the appropriate track ID.

Reply:
Done

Reply:
I confused it with something else. Fixed. Thanks for pointing it out